src/Chapter01/TopLevelStatementWrapperForListing01.01.cs

```diff
@@ -0,0 +1,16 @@
+// See https://aka.ms/new-console-template for more information
```

Comment:
```suggestion
```

Reply:
This was generated by the project so I suspect we should leave it.